src/functions.sh

```diff
@@ -175,17 +175,16 @@ cmd_wrapper(){
     ERRLOG="sqm_error"
     if [ "$SILENT" -eq "1" ]; then
         ERRLOG="sqm_debug"
-        sqm_debug "cmd_wrapper: ${CALLERID}: invocation silenced by request, failure either expected or acceptable."
+        sqm_debug "cmd_wrapper: ${CALLERID}: invocation silenced by request, FAILURE either expected or acceptable."
```

Comment:
Why the YELLING? :)

Reply:
Ah, because the log output with failures is prefixed with FAILURES, capitalizing it in the help text made it easier to connect the two in my admittedly brief testing. But if you believe this to be too much, I am happy to tone this down again.

Reply:
I have not touched that yet since I find this is a tad easier to parse visually. I am happy to change this though if my rationale does not convince you, just let me know ;)